forge-game/src/main/java/forge/game/cost/CostExile.java

```diff
@@ -217,7 +217,6 @@ else if (type.contains("FromTopGrave")) {
 
         if (totalCMC) {
             int needed = Integer.parseInt(this.getAmount().split("\\+")[0]);
-            if (list.size() < needed) return false;
```

Comment:
I removed this line, because I think its possible in totalCMC to require an amount higher than the amount of cards available, because its comparing CMC not just amount of cards.